test/e2e/rollback_test.go

```diff
@@ -0,0 +1,54 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+var _ = Describe("NodeNetworkState controller", func() {
+	var (
+		br1Absent = nmstatev1alpha1.State(`interfaces:
+  - name: br1
+    type: linux-bridge
+    state: absent
+`)
+
+		br1 = nmstatev1alpha1.State(`interfaces:
+  - name: br1
+    type: linux-bridge
+    state: up
+    bridge:
+      options:
+        stp:
+          enabled: false
+      port: []
+`)
+	)
+	Context("when there is an error at vlan-filtering", func() {
+		BeforeEach(func() {
+			By("Rename vlan-filtering to force failure after nmstatectl set")
+			runAtPods("sudo", "mv", "/usr/local/bin/vlan-filtering", "/usr/local/bin/vlan-filtering.bak")
+		})
+		AfterEach(func() {
+			By("Correct the vlan-filtering script name")
```

Comment:
This tells me we are fixing something but you never said we are breaking the name. Would you either fix this `By` or the one in `BeforeEach`?

Reply:
Let's reworkd the By